src/config/settings.ts

```diff
@@ -36,7 +36,13 @@ export let settings: Settings = {
     solana:
       "https://quiet-solemn-asphalt.solana-mainnet.quiknode.pro/91aa5918df65520d47a54d88590f1503a4e804b1",
     dogecoin: "https://go.getblock.io/5f7f5fba970e4f7a907fcd2c5f4c38a2",
-    turbochain: "https://rpc-0x4e45415f.aurora-cloud.dev ",
+    turbochain: "https://rpc-0x4e45415f.aurora-cloud.dev",
+  },
+  /**
+   * SiloToSilo addresses on blockchains within Aurora Engine
+   */
+  silo: {
+    turbochain: "", // TODO: add address to deployed contract at turbochain
```

Comment:
I doubt about having a config for this, as it's not expected to be changed as it's bound to a chain. I would suggest moving it to just constants.